config/manager/manager.yaml

```diff
@@ -1,75 +1,50 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: ibm-block-csi-operator
-  namespace: default
   labels:
-    product: ibm-block-csi-driver
-    csi: ibm
-    app.kubernetes.io/name: ibm-block-csi-operator
     app.kubernetes.io/instance: ibm-block-csi-operator
     app.kubernetes.io/managed-by: ibm-block-csi-operator
+    app.kubernetes.io/name: ibm-block-csi-operator
+    csi: ibm
+    product: ibm-block-csi-driver
+  name: ibm-block-csi-operator
+  namespace: default
 spec:
   replicas: 1
   selector:
     matchLabels:
       app.kubernetes.io/name: ibm-block-csi-operator
   template:
     metadata:
+      annotations:
+        productID: 5027566ef6c54de49028be7df25119e1
+        productMetric: FREE
+        productName: IBM Block CSI Driver
+        productVersion: 1.8.0
       labels:
-        app.kubernetes.io/name: ibm-block-csi-operator
         app.kubernetes.io/instance: ibm-block-csi-operator
         app.kubernetes.io/managed-by: ibm-block-csi-operator
-        product: ibm-block-csi-driver
+        app.kubernetes.io/name: ibm-block-csi-operator
```

Comment:
most of the diff looks like "rearranging lines", can we avoid it so the diff will be minimal?

Reply:
I needed to do it since our yamls weren't aligned and that the CI won't fail, it will be easier for you if I will separate the PR's? one PR will be the yamls diffs and the other one will be the scripts that check it

Reply:
yes, that would be great!

Reply:
I opened two PRs, this is how you wanted it?
https://github.com/IBM/ibm-block-csi-operator/pull/227
https://github.com/IBM/ibm-block-csi-operator/pull/226